### ISSUE TITLE
Document actor events better

### ISSUE THREAD

**Issue by fnesveda** [apify/apify-docs#283]
There are some events being sent to the actor runs over websocket about what is happening with the actor run (run is aborting, migrating etc.). We have them described a bit in the [SDK docs](https://sdk.apify.com/docs/api/apify#apifyevents), but the documentation is outdated. We should update the docs and move them to the general actor docs, because the events are not specific just to Javascript actors, any actor can use them.

**Comment by mtrunkat** [comment]
Good point, I added a page in the platform docs - https://docs.apify.com/platform/actors/development/programming-interface/system-events

So assigning `t-tooling` to update SDK docs and link this from there to have it centralized at a single place.

**Comment by TC-MO** [comment]
Closing as part of a docs backlog cleanup. This issue has been open for 6+ months without activity. If the work is still relevant, feel free to reopen or create a new issue with updated context.
